simulator/simulator.py

```diff
@@ -113,14 +133,25 @@
 sensor.register_sequence_executor(sequence_executor)
 ft6x36.register_sequence_executor(sequence_executor)
 
+##################################
+# Handle absolute path for files #
+##################################
```

Comment:
Why this box, it make this looks like docs but it is a comment?

Reply:
Put this only to clarify that for the simulator to work on calls in the `current working directory`, the full path was needed